itsy-bitsy-data-structures.js

```diff
@@ -748,8 +748,7 @@ class Stack {
  * rather than the end. Removing the oldest items rather than the most recent.
  *
  * Again, because this limits the amount of functionality, there are many
- * different ways of implementing it. A good way might be to use a linked list
- * which we will see later.
+ * different ways of implementing it. A good way might be to use a list.
```

Comment:
@Kikobeats In this comment he isn't referring to the current implementation which is indeed using list and is O(N) but instead clarifying that a good way to implement them would be a linked list. Although, the sentence itself is ambiguous and can be clarified if the author wants.